gcsfs/dask_link.py

```diff
@@ -1,5 +1,10 @@
+import logging
+
 from .core import GCSFileSystem
 
+logging.basicConfig()
+logger = logging.getLogger('dask_link')
```

Comment:
"gcsfs" - this is then consistent throughout the package

Reply:
Done.